### PR TITLE
VZ-9655 show cloud credentials error on Add New template of rancher-UI 

### PR DESCRIPTION
**Pull request by knelson7**

### CODE REVIEW

File: translations/en-us.yaml [hunk 1]
```diff
@@ -8569,6 +8569,7 @@ modalAddCloudKey:
   saving:
     validating: Validating Keys
   errors:
+    errorMessage: 'There was a problem validating your credentials. Please try again.'
```
Comment (jmaron99) [inline translations/en-us.yaml]: perhaps this should be more descriptive - 'authValidationError'?

File: lib/global-admin/addon/components/cru-cloud-credential/component.js [hunk 2]
```diff
@@ -380,9 +382,11 @@ export default Component.extend(ViewNewEdit, {
   },
 
   setError(message = '') {
```
Comment (jmaron99) [inline lib/global-admin/addon/components/cru-cloud-credential/component.js:382]: it looks like this method is called from an OCI auth validation failure with an error message. Is it empty?  Is it possibly more descriptive then the standard message that is in the translation bundle?  Was it too descriptive and deemed inappropriate to display?

File: lib/global-admin/addon/components/cru-cloud-credential/component.js [hunk 3]
```diff
@@ -380,9 +382,11 @@ export default Component.extend(ViewNewEdit, {
...
+    set(this, 'cloudErrorExists', true);
     const translation = this.intl.t('modalAddCloudKey.errors.validation', { status: message });
 
     set(this, 'validatingKeys', false);
+    set(this, 'cloudError', this.intl.t('modalAddCloudKey.errors.errorMessage'));
```
Comment (anders-swanson) [inline lib/global-admin/addon/components/cru-cloud-credential/component.js]: this `setError` function is supposed to be a generic error setting function. I think we should be not modifying this function (only calling it with the correct error message on L360)

File: lib/global-admin/addon/components/cru-cloud-credential/component.js [hunk 4]
```diff
@@ -380,9 +382,11 @@ export default Component.extend(ViewNewEdit, {
...
 
     this.parseAndCollectErrors(translation, true);
```
Comment (anders-swanson) [inline lib/global-admin/addon/components/cru-cloud-credential/component.js]: looks like they didn't implement this function. We should fix this to set the error

File: lib/global-admin/addon/components/cru-cloud-credential/component.js (hunk 2 above)
Comment (knelson7) [reply lib/global-admin/addon/components/cru-cloud-credential/component.js:384]: It was too descriptive and we decided the general one was better.

Comment (knelson7) [reply lib/global-admin/addon/components/cru-cloud-credential/component.js:382]: It's not empty. It is pretty descriptive but we can show it. 
The error for the specific case you filed is 
"can not create client, bad configuration: PEM data was not found in buffer: There was a problem validating your keys. Enter valid credentials and try again."

File: translations/en-us.yaml (hunk 1 above)
Comment (knelson7) [reply translations/en-us.yaml]: Restructured code to use existing error display setup so this is actually no longer needed.

**Review by jmaron99 — APPROVED** [review]

**Review by anders-swanson — APPROVED** [review]

**Pull request merged**